runtime/moonbase/src/xcm_config.rs

```diff
@@ -319,7 +319,7 @@ impl pallet_xcm::Config for Runtime {
 	type SendXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
 	type XcmRouter = XcmRouter;
 	type ExecuteXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
-	type XcmExecuteFilter = Nothing;
+	type XcmExecuteFilter = Everything;
```

Comment:
Seems like a big change to allow one call, although this seems to be the pattern in e.g. the polkadot codebase...

Reply:
Yeah, that's what it seems